feature_selection/forward_selection.py

```diff
@@ -36,8 +36,7 @@ def forward_selection(scorer, X, y, min_features=1, max_features=10):
     >>> from sklearn.datasets import make_friedman1
     >>> data, target = make_friedman1(n_samples=200, n_features=15,
     >>> random_state=0)
-    >>> from feature_selection.forward_selection \
-    >>>      import forward_selection
+    >>> from feature_selection import forward_selection
```

Comment:
Actually, I think it's correct the way it was. We are investigating this now. It is related to whether the appropriate imports are in the `__init__.py` file but they are currently causing the build to fail.

Reply:
Ok, sounds good Ryan.
Just when accepting or rejecting the PR, please take into consideration that these are two changes and not only one:
1) in the name of the example of the function, and
2) erase the backslash (“\”) that was affecting the example in the docs (as you can see in the image  the import `forward_selection` got at the right after the `>>>`)
<img width="700" alt="image" src="https://user-images.githubusercontent.com/53915521/76689402-d6e45280-65f2-11ea-8770-8f826b1de41c.png">

Reply:
No worries. Once I merge this PR, that will be fixed.